IMAGES.rst

```diff
@@ -0,0 +1,389 @@
+
+The command that build the production image are optimised for size of the image.
+
+In Breeze by default, the images are built using local sources of Apache Airflow. However
+you can also build production images from github sources - providing ``--install-airflow-version``
+parameter to Breeze. This will install airflow inside the production image using sources downloaded from
+specified tag or branch. Internally airflow will be installed using the command:
+
+.. code-block::
+
+    pip install https://github.com/apache/airflow/archive/<tag>>.tar.gz#egg=apache-airflow \
+       --constraint https://raw.githubusercontent.com/apache/airflow/<tag>/requirements/requirements-python3.7.txt
+
+
+Technical details of Airflow images
+===================================
+
+The CI image is used by Breeze as shell image but it is also used during CI builds on Travis.
+The image is single segment image that contains Airflow installation with "all" dependencies installed.
+It is optimised for rebuild speed (AIRFLOW_CONTAINER_CI_OPTIMISED_BUILD flag set to "true").
+It installs PIP dependencies from the current branch first - so that any changes in setup.py do not trigger
+reinstalling of all dependencies. There is a second step of installation that re-installs the dependencies
+from the latest sources so that we are sure that latest dependencies are installed.
+
+The production image is a multi-segment image. The first segment "airflow-build-image" contains all the
+build essentials and related dependencies that allow to install airflow locally. By default the image is
+build from a released version of Airflow from Github, but by providing some extra arguments you can also
+build it from local sources. This is particularly useful in CI environment where we are using the image
+to run Kubernetes tests. See below for the list of arguments that should be provided to build
+production image from the local sources.
+
+Note! Breeze by default builds production image from local sources. You can change it's behaviour by
+providing ``--install-airflow-version`` parameter, where you can specify the
+tag/branch used to download Airflow package from in github repository. You can
+also change the repository itself by adding --dockerhub-user and --dockerhub-repo flag values.
+
+Manually building the images
+----------------------------
+
+You can build the default production image with standard ``docker build`` command but they will only build
+default versions of the image and will not use the dockerhub versions of images as cache.
+
+
+CI images
+.........
+
+The following arguments can be used for CI images:
+
++------------------------------------------+------------------------------------------+------------------------------------------+
+| Environment variable                     | Default value                            | Description                              |
```

Comment:
```suggestion
| Build arg                     | Default value                            | Description                              |
```

Since these are controlled by `docker build --build-arg` aren't they?

Reply:
Or `Argument` as you have it in the next block.

Reply:
Yep!